src/pool/srv_pool.c

```diff
@@ -4396,7 +4396,7 @@ pool_svc_update_map(struct pool_svc *svc, crt_opcode_t opc, bool exclude_rank,
 	bool				updated;
 	int				rc;
 	char				*env;
-	uint64_t			delay = 0;
+	uint64_t			delay = 2;
```

Comment:
what's the purpose of this delay?

Reply:
to avoid conflict with pool map refresh (IV sync), otherwise it will need retry anyway.